src/wasm-lib/kcl/src/ast/types.rs

```diff
@@ -850,6 +909,18 @@ impl BinaryPart {
         }
     }
 
+    pub fn module_id(&self) -> ModuleId {
```

Comment:
IMO all these `module_id()` methods on AST nodes should be in their own new file, like we did for digests and execution.

Reply:
Any preference on whether to move only `module_id()` or all of `start()`, `end()`, and `module_id()`?

Reply:
I did the simplest thing.